README.md

```diff
@@ -29,6 +29,7 @@ The solutions are best read in the following order:
   - [__await.js__](./await.js)
   - [__tasks.js__](./tasks.js)
   - [__lazy-either.js__](./lazy-either.js)
+  - [__most.js__](./most.js)
```

Comment:
Let's group this with the other Promise-based solutions (after __promises-ramda.js__, say).